### DISCUSSION TITLE
mspider/user-guide/mesh-config/component-hpa/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#3774] [Announcements]
# mspider/user-guide/mesh-config/component-hpa/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/mspider/user-guide/mesh-config/component-hpa/

**Comment by kebe7jun** [comment]
>  ![环境依赖](https://docs.daocloud.io/daocloud-docs-images/docs/mspider/user-guide/mesh-config/images/meshrcfg07.png)

图片无法显示。

**Reply by Adityayxt** [reply]
已修复
